### PR TITLE
Add test app

### PR DESCRIPTION
**Pull request by Shan1024**
This commit will add a new app which can be used to test fragments.

### CODE REVIEW

File: components/pom.xml [hunk 1]
```diff
@@ -15,14 +15,12 @@
...
 
-<project xmlns="http://maven.apache.org/POM/4.0.0"
-         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
     <groupId>org.wso2.carbon.uuf.common</groupId>
     <artifactId>components-parent</artifactId>
-    <version>1.0.0-SNAPSHOT</version>
```
Comment (this) [inline components/pom.xml]: Revert all the changes in this file.

File: pom.xml [hunk 2]
```diff
@@ -15,14 +15,12 @@
   ~ limitations under the License.
 -->
 
-<project xmlns="http://maven.apache.org/POM/4.0.0"
```
Comment (this) [inline pom.xml]: Revert this change.

File: pom.xml [hunk 3]
```diff
@@ -15,14 +15,12 @@
...
-         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
     <groupId>org.wso2.carbon.uuf.common</groupId>
     <artifactId>common-parent</artifactId>
-    <version>1.0.0-SNAPSHOT</version>
```
Comment (this) [inline pom.xml]: Revert this change.

File: themes/org.wso2.carbon.uuf.common.theme.default/pom.xml [hunk 4]
```diff
@@ -15,14 +15,12 @@
   ~ limitations under the License.
 -->
 
-<project xmlns="http://maven.apache.org/POM/4.0.0"
```
Comment (this) [inline themes/org.wso2.carbon.uuf.common.theme.default/pom.xml]: Revert all the changes in this file.

File: themes/pom.xml [hunk 5]
```diff
@@ -15,14 +15,12 @@
   ~ limitations under the License.
 -->
 
-<project xmlns="http://maven.apache.org/POM/4.0.0"
```
Comment (this) [inline themes/pom.xml]: Revert all the changes in this file.

**Review by this — CHANGES_REQUESTED** [review]

**Review by this — APPROVED** [review]

File: tests/apps/org.wso2.carbon.uuf.common.test.app/src/main/pages/index.hbs [hunk 6]
```diff
@@ -0,0 +1,6 @@
+{{layout "org.wso2.carbon.uuf.common.foundation.main"}}
+{{title "Test| " @config.appName}}
+{{#fillZone "content"}}
+    <h1>Hello World!</h1>
+{{/fillZone}}
+{{fragment "org.wso2.carbon.uuf.common.foundation.uuf-client"}}
```
Comment (this) [inline tests/apps/org.wso2.carbon.uuf.common.test.app/src/main/pages/index.hbs]: Move this line to line no 3 (before #fillZone)

**Review by this — CHANGES_REQUESTED** [review]

**Review by this — APPROVED** [review]

File: pom.xml [hunk 7]
```diff
@@ -45,6 +45,7 @@
     <modules>
         <module>components</module>
         <module>themes</module>
+        <module>tests</module>
```
Comment (Kishanthan) [inline pom.xml]: if this is not used for testing purpose (i.e writing test artifacts and samples) then this module should be names as samples

**Review by Kishanthan — CHANGES_REQUESTED** [review]

**Review by Kishanthan — APPROVED** [review]

**Pull request merged**